Description: Fix compilation errors caused by unmappable characters for encoding ASCII
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java
+++ b/src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java
@@ -569,7 +569,7 @@
      * </tr>
      * <tr>
      * <td>IBM 1.5 (French JVM)</td>
-     * <td>javadoc version complète de "J2RE 1.5.0 IBM Windows 32 build pwi32pdev-20070426a"</td>
+     * <td>javadoc version complete de "J2RE 1.5.0 IBM Windows 32 build pwi32pdev-20070426a"</td>
      * </tr>
      * <tr>
      * <td>FreeBSD 1.5</td>
